starfish/data/__init__.py

```diff
@@ -0,0 +1,35 @@
+from starfish import Experiment
+
+
+def MERFISH(test_data: bool=False):
```

Comment:
+1 to @ttung comment that we should make this `use_test_data`. 

Also made #648

Reply:
https://github.com/spacetx/starfish/pull/651